src/error-utils/index.js

```diff
@@ -0,0 +1,92 @@
+import { isEmpty } from 'lodash/fp';
+import joi from 'joi';
+
+const errorDetailsEntrySchema = joi
+  .object()
+  .keys({
+    field: joi.string().allow('', null),
```

Comment:
Why are we allowing empty `strings` and `null`?

Reply:
Some Mongodb validation errors don't include field and values. Others come with null or empty string. Rather than force the user to allows pass empty string when such things happens, it makes sense to allow them